### PR TITLE
enable rockcomp in flow

### PR DESCRIPTION
**Pull request by totto82**
Depends on OPM/opm-models#554

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this opm-models=554 please

**Comment by totto82** [comment]
benchmark please

**Comment by ytelses** [comment]
Benchmark result overview: 




| Test | Configuration | Relative |
| ---- | ------------- | -------- |
* Speed-up = Total time master / Total time pull request. Above 1.0 is an improvement. *



View result details @ https://www.ytelses.com/opm/?page=result&id=551

**Comment by totto82** [comment]
benchmark opm-models=554 please

File: ebos/eclproblem.hh [hunk 1]
```diff
@@ -1765,7 +1763,7 @@ public:
     template <class LhsEval>
     LhsEval rockCompPoroMultiplier(const IntensiveQuantities& intQuants, unsigned elementIdx) const
     {
-        if (!enableExperiments || rockCompPoroMult_.size() == 0)
+        if (rockCompPoroMult_.size() == 0)
```
Comment (akva2) [inline ebos/eclproblem.hh]: .empty()

File: ebos/eclproblem.hh [hunk 2]
```diff
@@ -1797,7 +1795,7 @@ public:
     template <class LhsEval>
     LhsEval rockCompTransMultiplier(const IntensiveQuantities& intQuants, unsigned elementIdx) const
     {
-        if (!enableExperiments || rockCompTransMult_.size() == 0)
+        if (rockCompTransMult_.size() == 0)
```
Comment (akva2) [inline ebos/eclproblem.hh]: .empty()

File: ebos/eclproblem.hh [hunk 3]
```diff
@@ -1828,7 +1826,7 @@ public:
      */
     Scalar overburdenPressure(unsigned elementIdx) const
     {
-        if (!enableExperiments || overburdenPressure_.size() == 0)
+        if (overburdenPressure_.size() == 0)
```
Comment (akva2) [inline ebos/eclproblem.hh]: .empty()

**Comment by dr-robertk** [comment]
@totto82: Could you address the comments. Use method empty instead of size() == 0.

**Comment by dr-robertk** [comment]
jenkins build this opm-models=554 please

**Comment by blattms** [comment]
backported to release via #2070

**Pull request merged**